client/me/help/help-contact/index.jsx

```diff
@@ -518,7 +540,7 @@ const HelpContact = React.createClass( {
 					showSubjectField: false,
 					showHowCanWeHelpField: false,
 					showHowYouFeelField: false,
-					showSiteField: hasMoreThanOneSite,
+					showSiteField: false,
```

Comment:
Is this change intentional? Seems like we're updating Kayako functionality here and maybe introducing a regression.